src/vs/platform/terminal/common/capabilities/commandDetectionCapability.ts

```diff
@@ -674,7 +674,7 @@ function getOutputMatchForCommand(executedMarker: IMarker | undefined, endMarker
 			i = wrappedLineStart;
 			lines.unshift(getXtermLineContent(buffer, wrappedLineStart, wrappedLineEnd, cols));
 			if (!match) {
-				match = lines[0].match(matcher);
+				match = lines.join('\n').match(matcher);
```

Comment:
We didn't want to match against all lines joined together, the idea when we last discussed this was to let the quick fix handle multiple lines after the initial match. Doing `lines.join('\n')` will add pretty significant overhead as well.

Reply:
You're right, but we mandate at least one match found in the output currently in order to return lines. All of the other quick fixes have output matchers that match a wrapped line

Reply:
> All of the other quick fixes have output matchers that match a wrapped line

You mean do not right? We want `GitSimilarOutputRegex` to only match `most similar (command|commands) (is|are)` now

Reply:
Yes, so we should update that and the builtin quick fix. It's just a little weird because then `outputMatch` doesn't mean it's an actual match / what we're looking for anymore.

Reply:
It's still a match, just a single line match. The experiment trying to have some match multiple lines didn't really work due to perf issues (which is what the regex's are meant to prevent in the first place), too many unique strings being constructed.

Reply:
I'll update this to make that clear
![Screenshot 2023-03-08 at 9 49 13 AM](https://user-images.githubusercontent.com/29464607/223761307-d236ddec-c3fe-496d-8222-cf42206b6ce0.png)

Reply:
actually, I guess that doesn't need to be updated 🤔 . I just find it a little confusing still. Will think about if it can be improved

Reply:
We can clarify it only matches single lines

Reply:
done